ci/run_tests.sh

```diff
@@ -21,6 +21,18 @@ export PYTHONUNBUFFERED=1
 # Add the path where pip installs commands to PATH.
 export PATH=$PATH:${HOME}/.local/bin
 
+# Copy docfx to a local directory, so that we'll be able
```

Comment:
This looks very hacky.

Reply:
Oh it's *absolutely* hacky. I'm scoping out the boundaries of what's at least feasible :)